ironic/drivers/modules/redfish/utils.py

```diff
@@ -274,3 +278,100 @@ def _get_system():
                       'node %(node)s. Error: %(error)s',
                       {'address': driver_info['address'],
                        'node': node.uuid, 'error': e})
+
+def _ping_host(host):
+    response = subprocess.call(
+        "ping -c 1 {} 2>&1 1>/dev/null".format(host), shell=True)
+    return (response == 0)
+
+def wait_for_host_state(host,
+                        alive=True,
+                        ping_count=3,
+                        retries=24):
+    if alive:
+        ping_type = "pingable"
+
+    else:
+        ping_type = "not pingable"
+
+    LOG.info("Waiting for the iDRAC to become %s", ping_type)
```

Comment:
Need to include the IP of the iDRAC in this logging statement

Reply:
Done, Logged iDRAC IP address while logging.